### PR TITLE
adding support for the memcached resource type in local racks

### PR DESCRIPTION
**Pull request by camerondgray**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ coverage.txt
 profile.out
 /monitor
 /rack
+.idea/
```
Comment (ddollar) [inline .gitignore]: User-specific ignores like this should be in your user-level .gitignore.

File: provider/local/converge.go [hunk 2]
```diff
@@ -291,6 +291,8 @@ func resourcePort(kind string) (int, error) {
 		return 5432, nil
 	case "redis":
 		return 6379, nil
+	case "memcached":
```
Comment (ddollar) [inline provider/local/converge.go]: Please put these case statements in alphabetical order.

File: provider/local/converge.go [hunk 3]
```diff
@@ -304,6 +306,8 @@ func (p *Provider) resourceURL(app, kind, name string) (string, error) {
 		return fmt.Sprintf("postgres://postgres:password@%s.resource.%s.%s:5432/app?sslmode=disable", name, app, p.Rack), nil
 	case "redis":
 		return fmt.Sprintf("redis://%s.resource.%s.%s:6379/0", name, app, p.Rack), nil
+	case "memcached":
+		return fmt.Sprintf("%s.resource.%s.%s:11211", name, app, p.Rack), nil
```
Comment (ddollar) [inline provider/local/converge.go]: This should probably start with memcached:// for consistency with the other resources.

**Comment by camerondgray** [comment]
@ddollar PR is updated. Let me know if you see any new issues.

File: .gitignore [hunk 4]
```diff
@@ -2,4 +2,4 @@
 coverage.txt
 profile.out
 /monitor
-/rack
+/rack
```
Comment (ddollar) [inline .gitignore]: Can you please remove this change, might need to configure your editor to allow newlines at the end of a file.

**Comment by camerondgray** [comment]
apologies. Should be all reverted now

**Pull request closed without merging**